src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/PageTypeSelector/PageTypeSelector.jsx

```diff
@@ -33,6 +33,32 @@ class PageTypeSelector extends Component {
         const hierarchy = this._getHierarchyLabel();        
         const components = this.getComponents(); 
 
+        let pageParentInfoStyle = null;
+        let parentPageStyleLabel = null;
+        let parentPageStyleContent = null;
+        
+
+        if (hierarchy.length < 31) {
```

Comment:
Why did you choose 31 as the cutoff length? Keeping in mind that the font is variable width, is it low enough to give a desired effect when the letters are wider on average?

Reply:
Agree on that. I will change to find out the width if this field